packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -882,6 +889,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this._summarizer;
     }
 
+    public get summariesDisabled(): boolean {
+        return this.runtimeOptions.summaryOptions.generateSummaries === false;
```

Comment:
To be updated after your other PR (server configs?)

Reply:
Right. I have a few conflicts to resolve.